t2kdm/interactive.py

```diff
@@ -156,7 +156,7 @@ def ls(remotepath, *args, **kwargs):
         entries = dm.iter_ls(remotepath, *args, **kwargs)
     else:
         entries = dm.iter_ls_se(remotepath, *args, se=se, **kwargs)
-    if long:
+    if int:
```

Comment:
This is defintely a wrong change. `long` is a variable name.

Reply:
If `long` is also a variable type in Python, maybe we should rename the variable.

Reply:
It seems it is for python3... 😅

Reply:
Changed it to long_str